.travis.yml

```diff
@@ -4,8 +4,9 @@ services:
 script:
 - docker build --target test --tag my-test-image .
 - docker run my-test-image todo_app/items_test.py    
-- docker run -e TOKEN -e KEY -e TOID -e DOINGID -e DONE -e CLIENT -e GITHUBID -e WEBAPPLICATIONCLIENT -e CLIENTSECRET -e TODOBOARD -e DOINGBOARD -e DONEBOARD -e CONNECTIONSTRING my-test-image  todo_app/e2e_test.py 
+- docker run -e TOKEN -e KEY -e TOID -e DOINGID -e DONE -e CLIENT -e GITHUBID -e WEBAPPLICATIONCLIENT -e CLIENTSECRET -e TODOBOARD -e DOINGBOARD -e DONEBOARD -e CONNECTIONSTRING -e WEBHOOK_URL my-test-image  todo_app/e2e_test.py 
```

Comment:
You shouldn't need to pass the `WEBHOOK_URL` into your e2e test

Reply:
Updated, thank you